### PR TITLE
Multi pipeline

### PR DESCRIPTION
**Pull request by RachelRen05**
@LewisLiuPub 
This PR adds multiple pipelines description and sample. Please have a review. Thank.

### CODE REVIEW

File: dynamic_vino_lib/include/dynamic_vino_lib/inputs/standard_camera.hpp [hunk 1]
```diff
@@ -21,7 +21,15 @@
...
 #include <opencv2/opencv.hpp>
+
+
 #include "dynamic_vino_lib/inputs/base_input.hpp"
+#include <stdio.h>
+#include <linux/videodev2.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/ioctl.h>
+
```
Comment (LewisLiuPub) [inline dynamic_vino_lib/include/dynamic_vino_lib/inputs/standard_camera.hpp]: duplicated with PR#107?

File: dynamic_vino_lib/src/inputs/standard_camera.cpp [hunk 2]
```diff
@@ -18,15 +18,26 @@
...
 {
-  static int camera_count_ = 0;
-  setInitStatus(cap.open(camera_count_));
-  setWidth((size_t)cap.get(CV_CAP_PROP_FRAME_WIDTH));
-  setHeight((size_t)cap.get(CV_CAP_PROP_FRAME_HEIGHT));
-  camera_count_++;
-  return isInit();
+  static int camera_count_ = -1;
+  int fd; // A file descriptor to the video device
+  struct v4l2_capability cap;
```
Comment (LewisLiuPub) [inline dynamic_vino_lib/src/inputs/standard_camera.cpp]: @luyang00 this part is not compatible with non-linux OS. and a MACRO to only enable it for Linux?

Comment (luyang00) [reply dynamic_vino_lib/src/inputs/standard_camera.cpp]: thank you for your suggestion, v4l2 does only work for linux, I will add a MACRO later.

Comment (LewisLiuPub) [reply dynamic_vino_lib/src/inputs/standard_camera.cpp]: Ok, I'll merge the code first, but I'll file a bug to record the fixing process in parallel.

**Pull request merged**